scripts/update-index.sh

```diff
@@ -5,28 +5,43 @@ fail() {
     exit 1
 }
 
+check_package() {
+    [ -d "package/$1" ] || fail "package/$1 directory does not exist... run package.sh first"
+}
+
+update_index() {
+    # Merge new charts into current index
+    : ${GH_RELEASES_URL:="https://github.com/nuodb/nuodb-helm-charts/releases/download"}
+    helm repo index "package/$1" --merge "$1/index.yaml" --url "$GH_RELEASES_URL"
```

Comment:
Nit: As `$1` is used multiple times in this function, it worth assigning it to a variable for better readability.